tin/templates/assignments/manage_student.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block main %}
+  <h2>Managing Student {{ user.username }}</h2>
```

Comment:
Would probably be helpful to have "current quota" metrics when changing a students caps. For example, something like "Submissions used: 12/15. Submissions used after due date: 0/0" to help inform the override